specification/paths/Brokers-broker_id-carriers.json

```diff
@@ -0,0 +1,109 @@
+{
+  "parameters": [
+    {
+      "$ref": "#/components/parameters/path-broker_id"
+    }
+  ],
+  "get": {
+    "tags": [
+      "Brokers"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "enterprise.manage"
+        ]
+      }
+    ],
+    "summary": "Get the available carriers for a specific broker.",
+    "description": "This endpoint retrieves available carriers for a broker with the supplied id.",
+    "responses": {
+      "200": {
+        "description": "Retrieved the available carriers for the broker.",
+        "content": {
+          "application/vnd.api+json": {
+            "schema": {
+              "type": "object",
+              "required": [
+                "data"
+              ],
+              "additionalProperties": false,
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "allOf": [
+                      {
+                        "$ref": "#/components/schemas/Carrier"
+                      },
+                      {
+                        "required": [
+                          "id",
+                          "links"
+                        ],
+                        "properties": {
+                          "attributes": {
+                            "required": [
+                              "status",
+                              "currency",
+                              "name",
+                              "created_at"
+                            ]
+                          },
+                          "relationships": {
+                            "properties": {
+                              "owner": {
+                                "example": {
+                                  "links": {
+                                    "related": "$API_HOST/carriers/9cdf86e8-333f-4ed9-bb31-4935c780c947"
+                                  }
+                                }
+                              }
+                            }
+                          }
+                        }
+                      }
+                    ]
+                  }
+                },
+                "meta": {
+                  "$ref": "#/components/schemas/PaginationMeta"
+                },
+                "links": {
+                  "type": "object",
+                  "required": [
+                    "self",
+                    "first",
+                    "last"
+                  ],
+                  "properties": {
+                    "self": {
+                      "type": "string",
+                      "example": "API_HOST/organizations/9cdf86e8-333f-4ed9-bb31-4935c780c947/contracts?page[number]=3&page[size]=30"
```

Comment:
Can you please do a global search on `API_HOST` and make sure all of them are `$API_HOST`. I noticed the `$` is missing in multiple (old) files.